bitnami/odoo/ci/values-hpa-pdb.yaml

```diff
@@ -0,0 +1,4 @@
+autoscaling:
+  enabled: true
+pdb:
+  create: true
```

Comment:
I added it but I get errors when pushing, i.e.:

```
$ helm template --values bitnami/odoo/ci/values-metrics-and-ingress.yaml bitnami/odoo | kubeval --ignore-missing-schemas --strict --openshift failed.
...
templates/ingress.yaml contains an invalid Ingress - service: Additional property service is not allowed
```

So I am going to skip this.

Reply:
That's normal and it's due to some fields we're including in the **ingress.yaml** that **kubeval** doesn't recognize. If that's the only thing the GitHooks actions complain about, send the changes with `--no-verify` flag.
@javsalgar idk if this is something we want to revisit

Reply:
For now I'd avoid including such change because it is making the actual push process fail, and I'd not like a user to face such an issue when trying to contribute (it looks like something is wrong with the chart).